bundle/libraries/local_path.go

```diff
@@ -64,7 +64,8 @@ func IsLibraryLocal(dep string) bool {
 		return false
 	}
 
-	// If the dependency is a requirements file, it's not a valid local path
+	// If the dependency is a requirements file, it can either be a local path or a remote path.
+	// But we still don't treat it as a local library because we just translate this path in TranslatePaths mutator.
```

Comment:
I don't understand the second line comment.

Reply:
I guess should be rephrased that even though the path to requirements.txt is local we don't return true and treat this path as local library path but handle translation of these paths in separate code path in TranslatePath mutator